requirements.txt

```diff
@@ -7,7 +7,7 @@ pandas>=2.2.0 #latest sktime makes tests fail if pandas 2.1
 jinja2>=3
 scipy>=1.6.1,<=1.11.4 #to fix later (https://github.com/mljar/mljar-supervised/issues/691)
 joblib>=1.2.0,<1.4  # joblib<1.2.0 is vulnerable to Arbitrary Code Execution (https://github.com/advisories/GHSA-6hrg-qmvc-2xh8)  # joblib 1.4.0 breaks PyCaret
-scikit-learn>1.4.0 #scikit-learn 1.4.0 have a bug (https://github.com/scikit-learn/scikit-learn/pull/28241)
+scikit-learn>1.4.0,<1.5.0 #scikit-learn 1.4.0 have a bug (https://github.com/scikit-learn/scikit-learn/pull/28241)
```

Comment:
why doesn't pycaret support scikit-learn 1.5 ?

Reply:
Reverted thx for your comment